### ISSUE TITLE
Open by default in geddit 

### ISSUE THREAD

**Issue by erik9012** [kaangiray26/geddit-app#16]
Make it possible to open redddit domains within geddit by default. Just like aliucord has that options. This feature is useful for Android 11 and below and for people who rooted and disabled verified links  or for people using apps like linksheet 
![Screenshot_2023-07-16-13-41-06-375_com android settings](https://github.com/kaangiray26/geddit-app/assets/57501353/b3c2ab8e-8106-42a5-873b-a9896d89410a)

**Comment by erik9012** [comment]
![Uploading Screenshot_2023-07-16-13-44-33-526_com.android.settings.jpg…]()

**Comment by erik9012** [comment]
![Screenshot_2023-07-16-13-44-33-526_com android settings](https://github.com/kaangiray26/geddit-app/assets/57501353/3f915999-75ad-491b-9988-f755aaa32059)

**Comment by kaangiray26** [comment]
Hi,  
I've been researching how to do this automatically once you download the app. I've found the verified links added by the official Reddit app.

Here is a list of them:
```
amp.reddit.com
old.reddit.com
mod.reddit.com
accounts.reddit.com
new.reddit.com
click.redditmail.com
np.reddit.com
reddit.app.link
reddit.onelink.me
reddit-alternative.app.link
*.redd.it
reddit.com
m.reddit.com
www.reddit.com
s.reddit.com
```

**Comment by kaangiray26** [comment]
Normally, you would go to `Settings > Apps > Default apps > Opening links > Geddit > Add Link` to add custom URL schemes.

**Comment by kaangiray26** [comment]
I'll be needing lots of Reddit link samples :)

**Comment by kaangiray26** [comment]
# Samples

## User links
* https://www.reddit.com/user/kgb_26

## Subreddit links
* https://www.reddit.com/r/programming/
* https://www.reddit.com/r/programming/hot/
* https://www.reddit.com/r/programming/new/
* https://www.reddit.com/r/programming/rising/
* https://www.reddit.com/r/programming/top/
* https://www.reddit.com/r/programming/top/?t=all

## Post links
* https://www.reddit.com/r/webdev/comments/14wxuxj/ive_built_an_app_to_browse_reddit_without_their/

**Comment by erik9012** [comment]
Maybe it's possible to  do *.reddit.com, reddit.com/* to cover all in once

**Comment by erik9012** [comment]
*

**Comment by erik9012** [comment]
Can you enter a star before the urls to cover all in once

**Comment by kaangiray26** [comment]
> Can you enter a star before the urls to cover all in once

I'm not sure if Android intent-filters support those types of matchings but I'll try.

**Comment by kaangiray26** [comment]
I've added support for the links below and it will be available in v1.3.

```
amp.reddit.com
old.reddit.com
new.reddit.com
np.reddit.com
*.redd.it
reddit.com
m.reddit.com
www.reddit.com
sh.reddit.com
```

**Comment by kaangiray26** [comment]
A quick last note:

Although I've added support for these links, you must enable them by yourself in the settings. There will be a note in the README as well.
